### ISSUE TITLE
Errors on tables with no foreign keys

### ISSUE THREAD

**Issue by LiuDaveLiu** [mattbdean/Helium#38]
Here is the error from the [console](https://gist.github.com/LiuDaveLiu/fc0a23a6f40f8b8e8ceccb4725492c5b)

**Comment by mattbdean** [comment]
Can you reproduce the error with `yarn dev` for a better stacktrace?

**Comment by boazmohar** [comment]
[spinevis.int.janelia.org-1514326964502.log](https://github.com/mattbdean/Helium/files/1587966/spinevis.int.janelia.org-1514326964502.log)

**Comment by mattbdean** [comment]
Which table were you looking at? What's the URL?

**Comment by LiuDaveLiu** [comment]
We were looking at trial_note table. http://localhost:3000/tables/daveliu_map_experi/trial_note

**Comment by boazmohar** [comment]
You might not have permission to that table but I also get it with:
`tables/arseny_s1alm/behavior`

**Comment by mattbdean** [comment]
I saw the error last night but I'm not getting it anymore. Is there a process that reliably reproduces it?

**Comment by boazmohar** [comment]
For me I will see it if I navigate to a lot of tables, I just replicated it again but only after ~20 table lookups from the sidebar.
It might be related to #41, I am not sure but the table I got the error for (boazmohar_spineimaging/subject) if I go to the form I see this error:
`core.js:1427 ERROR TypeError: Cannot read property 'at' of undefined`
And I can see the values of the part table that are suppose to be hidden.

Tables that don't have the `at` error I don't see the part table maser keys.

Tell me if you want to Skype so we can be sure we are looking at the same thing.
